hack/update-godep-licenses.sh

```diff
@@ -30,6 +30,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+LANG=C
+LC_ALL=C
```

Comment:
If this is not exported, it isn't affecting the sort on line 124, is it?  sort without LC_ALL=C == future doom.